plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -614,7 +634,7 @@ def _decide_rpms_to_download(self, metadata_files, catalog):
             # scan through all the metadata to decide which packages to download
             package_info_generator = packages.package_list_generator(
                 primary_file_handle, primary.PACKAGE_TAG, primary.process_package_element)
-            wanted = self._identify_wanted_versions(package_info_generator)
+            wanted = self._identify_wanted_versions(package_info_generator, ids.TYPE_ID_RPM)
```

Comment:
I like this one more than other options. Good idea! (taking into account all the other weird options around). No changes to `self._identify_wanted_versions` sounds really good.